man/mtf-env-set.1

```diff
@@ -0,0 +1,20 @@
+.\" Copyright Petr Hracek, 2017
+.\"
+.\" This page is distributed under GPL.
+.\"
+.TH mtf-env-set 1 2017-11-01 "" "Linux User's Manual"
+.SH NAME
+mtf-env-set \- Part of Meta-Test-Family package. It prepares environment for testing containers.
+
+.SH SYNOPSIS
+\fBmtf-env-set prepares environment for testing containers
+
+.SH DESCRIPTION
+\fBmtf-env-set\fP is the binary file of Meta-Test-Family. It installs tests dependencies, docker service
+ and starts docker service.
+
+.SH NOTES
+\fBmtf-env-set\fP is useful for users who don't want to care about environment and their settings.
```

Comment:
this is a bit harsh, how about "mtf-env-set is useful for people who don't want to set the environment manually and wish to use this executable to do the job"